distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/solr-split.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "split-solr-app",
+  "description": "The Solr Split Catalog Provider (SCP) is an implementation of the CatalogProvider interface using Apache Solr ${solr.version} as a data store.\nThe Solr Split supports the same capabilities as the Solr Provider only allows further abilities to split into multiple solr collections via this single provider.",
+  "bundleLocations": [
+    "mvn:ddf.catalog.core/catalog-core-solr-defaultsolrcollectionprovider/${project.version}",
+    "mvn:ddf.catalog.core/catalog-core-provider/${project.version}"
```

Comment:
❓ These bundles match the feature definition for the `split-solr-provider` feature, not the `split-solr-app` feature. Is that what you intended?

Reply:
good catch, adding definition to match split-solr-app